src/style.css

```diff
@@ -0,0 +1,87 @@
+:root {
+  --color-light: #e1e7ed;
+  --color-dark: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.form:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.form_big:after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.form_small:after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.form input {
```

Comment:
Consider using classes instead of tag selectors

Reply:
<img width="610" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/70220667/41b6ef5a-282b-47a1-87d9-d805e651d2b1">
still not fixed

Reply:
I don't really understand, but I hope I did the right thing.